plugin/src/test/java/io/jenkins/plugins/casc/ConfigurationAsCodeTest.java

```diff
@@ -54,7 +57,12 @@ public void init_test_from_accepted_sources() throws Exception {
         // should be picked up
         Path target = Paths.get("jenkins.tmp");
         Path newLink = Paths.get(tempFolder.getRoot().getAbsolutePath(), "jenkins_5.yaml");
-        Files.createSymbolicLink(newLink, target);
+
+        try {
+            Files.createSymbolicLink(newLink, target);
+        } catch (IOException e) {
+            Assume.assumeFalse(Functions.isWindows());
```

Comment:
could you add an explanation on why please, something like:
```suggestion
            // This test often fails on windows for more details see: https://stackoverflow.com/questions/23217460/how-to-create-soft-symbolic-link-using-java-nio-files/24353758#24353758
            Assume.assumeFalse(Functions.isWindows());
```

Reply:
Message can be put directly to the `assumeFalse` method as a first argument